src/components/implementation/no_interface/scheddev/fprr/sl_mod_fprr.c

```diff
@@ -35,6 +38,7 @@ void
 sl_mod_block(struct sl_thd_policy *t)
 {
 	ps_list_rem_d(t);
+	sl_timeout_mod_block(sl_mod_thd_get(t), 0);
```

Comment:
It is setting timeout to the next period, if second parameter is 0, it uses thread's period as implicit timeout. I should add a comment here, I'll. I have a comment in the `sl_timeout_mod_block` function but not here. 

I did this in the module because in my understanding module should have more information about the threads and have wakeup/deadline etc intelligence built into it and not the sl core library.